### PR TITLE
[IMP] Update Travis for Transifex

### PR DESCRIPTION
**Pull request by pedrobaeza**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -15,21 +15,27 @@ python:
...
 install:
-  - git clone https://github.com/oca/maintainer-quality-tools.git ${HOME}/maintainer-quality-tools
-  #- git clone https://github.com/OCA/connector $HOME/connector -b ${VERSION}
+  - git clone --depth=1 https://github.com/oca/maintainer-quality-tools.git ${HOME}/maintainer-quality-tools
   - export PATH=${HOME}/maintainer-quality-tools/travis:${PATH}
   - travis_install_nightly
 
 script:
-  - travis_run_tests
+  - travis_after_tests_success
```
Comment (tafaRU) [inline .travis.yml]: It seems wrong, according to https://github.com/OCA/maintainer-quality-tools/blob/master/sample_files/.travis.yml#L78

Comment (pedrobaeza) [inline .travis.yml]: Sorry, I change the incorrect line

**Comment by pedrobaeza** [comment]
Fixed

**Comment by tafaRU** [comment]
Thanks :+1:

**Comment by antespi** [comment]
Thanks :+1:

**Comment by rafaelbn** [comment]
:+1: Happy Christmas!

**Pull request merged**